### DISCUSSION TITLE
Please add Gleam Language

### DISCUSSION THREAD

**Discussion by computerjohney** [codewars/codewars.com#3242] [Ideas]
Gleam is the new hotness. Its a typed functional language on the Beam VM (like Elixir, Erlang). See gleam.run
Exercism https://www.youtube.com/watch?v=yUPMVgvl4vo

**Comment by hobovsky** [comment]
New languages can be requested by creating a ticket in the [codewars/runner](https://github.com/codewars/runner/issues) repository. Please use the appropriate template and create a ticket there, providing all requested details.
